content/blog/2017/2017-04-10-jenkins-has-upgraded-to-java-8.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Starting with 2.54, Jenkins now requires Java 8"
+tags:
+- java8
+author: batmat
+---
+
+We link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced in January]
+that Jenkins would be upgrading its Java runtime dependency to Java 8 this
+year. After a sizable amount of preparation, this week's release of
+link:/changelog/#v2.54[Jenkins **2.54**] is the first weekly release to require
+a Java 8 runtime.
+
+For users of the _weekly_ release, this means that Jenkins 2.54 **must** have
+a Java 8 runtime installed on the system in order to
+run. Those using the
+link:https://hub.docker.com/r/jenkinsci/jenkins[jenkinsci/jenkins:latest]
+Docker container won't need to take any action, as the Java runtime environment
+is already bundled in the container.
+
+In addition to upgrading the Java Runtime Environment for the master, any
+connected **agents** must upgrade to a Java 8 runtime environment.
+
+The link:/changelog-stable[Long-Term Support] (LTS) release line however, has
+not yet been updated to require Java 8. We are expecting the first LTS release
+to require Java 8 in June.
+
+== Compatibility Notes
+
+**Using the Maven project type with Java 7**
+
+Users with jobs configured with the "Maven project" type may not be able to use
+Java 7 for their Maven jobs. The correct behavior is **not guaranteed** so
+proceed at your own risk. The Maven Project uses Jenkins Remoting to establish
+"interceptors" within the Maven executable. Because of this, Maven uses
+Remoting and other Jenkins core classes, and this behavior may break an update.
+
+See also:
+link:https://issues.jenkins-ci.org/browse/JENKINS-40990[JENKINS-40990].
```

Comment:
otherwise ypu can link to https://wiki.jenkins-ci.org/display/JENKINS/Maven+Project+Plugin ?
There is a warning about the compat issue and the link to 40990